repos/backend/endpoints/files.js

```diff
@@ -23,8 +23,8 @@ const saveFile = (app, config) => async (req, res) => {
 
 const loadFile = (app, config) => async (req, res) => {
   try {
-    const file = req.query.file
-    const meta = await getTestFile(config, file)
+    const path = req.query.path
```

Comment:
We should use a  name other then `path` here, because on `line 9` you're requiring and creating a `path` variable as well. It doesn't error, because of javascript scoping,  but can lead to odd bugs.

Maybe call it `filePath` here on line 26, or just reference it directly with `req.query.path` on line 27

Reply:
updated to `filePath`